model_hub/pyproject.toml

```diff
@@ -1,2 +1,43 @@
+[build-system]
+# Minimum requirements for the build system to execute.
+requires = ["setuptools>=64", "wheel", "setuptools_scm>=8"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "model-hub"
+description = "Model Hub for Determined Deep Learning Training Platform"
+requires-python = ">=3.6"
+
+authors = [
+  { name = "Determined AI", email = "ai-open-source@hpe.com" },
+]
+
+dependencies = [
+  "determined>=0.13.11",  # We require custom reducers for PyTorchTrial.
+]
+
+classifiers = [
+  "License :: OSI Approved :: Apache Software License",
+]
```

Comment:
Include license = {text = "Apache License 2.0"}?

Reply:
Sorry it took me so long to respond. `classifiers` is the current, PyPI-friendly method of specifying a license; as far as I can tell, it's only things like `pip show` that depend on that key now. I'm hesitant to add it back in if it's not necessary, but I will if you tell me to.

Reply:
Merging for now to get some stuff over the line and clean up branches, but let me know and I can open another PR to add the license string back.